README.md

```diff
@@ -1 +1,99 @@
-# MediaEditor-iOS
\ No newline at end of file
+# MediaEditor
+
+[![CircleCI](https://circleci.com/gh/wordpress-mobile/MediaEditor-iOS.svg?style=svg)](https://circleci.com/gh/wordpress-mobile/MediaEditor-iOS) [![Version](https://img.shields.io/cocoapods/v/MediaEditor.svg?style=flat)](http://cocoadocs.org/docsets/MediaEditor) [![License](https://img.shields.io/cocoapods/l/MediaEditor.svg?style=flat)](http://cocoadocs.org/docsets/MediaEditor) [![Platform](https://img.shields.io/cocoapods/p/MediaEditor.svg?style=flat)](http://cocoadocs.org/docsets/MediaEditor) [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+
+Media Editor is an open-source and extendable Swift-library that allows you to add the feature of editing images in your app. You can edit single or multiple images, from the device's library or any other source. It has been designed to feel natural and part of the OS.
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/7040243/73460209-dd767d80-4356-11ea-9a04-dcc6b0b5e233.PNG" width="340">
+</p>
+
+# Features
+
+- [x] `PHAsset` support
+- [x] Plain `UIImage` support
+- [x] Support to remote images
+- [x] Single media support
+- [x] Multiple media support
+- [x] Portrait and landscape
+- [x] Crop, zoom and rotate capability (thanks to [`TOCropViewController`](https://github.com/TimOliver/TOCropViewController))
+- [x] Easily extendable
+- [x] Customizable
+
+## Media Editor 101
```

Comment:
```suggestion
## Usage
```

Reply:
I feel like this section should be below installation. Thoughts?

Reply:
Since we have badges at the top that states that Carthage and Cocoapods are supported I feel like the user would be more interested in the syntax than how to install it.

Personally, I check the installation steps after I feel that I liked the library. And that includes the syntax.

Reply:
Makes sense. Leave it as it is. 👍